packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -594,7 +594,7 @@ class _GoogleMapState extends State<GoogleMap> {
 
 /// Builds a [MapConfiguration] from the given [map].
 MapConfiguration _configurationFromMapWidget(GoogleMap map) {
-  assert(!map.liteModeEnabled || Platform.isAndroid);
+  assert(!map.liteModeEnabled || isAndroid);
```

Comment:
If this is the only place we are checking platform, let's just drop the assert. It's fine to have it ignored silently instead of failing assertions, as it's documented that it's ignored on other platforms.

Reply:
done